aclogview/Enums/Combat.cs

```diff
@@ -111,3 +111,36 @@ public enum PowerBarMode {
     PBM_JUMP,
     PBM_DDD
 }
+
+public enum BodyPart
+{
+    UNDEFINED,
```

Comment:
These actually all need to shift by -1, so UNDEFINED = -1, HEAD = 0, CHEST = 1, etc. I would suggest just getting rid of the UNDEFINED entry.